src/component/Filter/Filter.tsx

```diff
@@ -0,0 +1,20 @@
+import { FilterProps } from "../../types/types";
+import styles from "./Filter.module.css";
+
+const Filter = ({ name, options, onChange }: FilterProps) => {
+  const optionElements = options.map((option) => {
+    return (
+      <option key={option.value} value={option.value}>
+        {option.label}
+      </option>
+    );
+  });
+
+  return (
+    <select className={styles.filter} name={name} onChange={onChange}>
+      {optionElements}
+    </select>
+  );
+};
```

Comment:
Filter 라기보단 Select 와 같이 정말 기본 컴포넌트 이름으로 네이밍 해보는 건 어떨까요?!
그리고 이런 컴포넌트의 경우에는 기본 select 속성을 props 에 추가해주면 사용처에서도 편할 것 같아요!

```suggestion

interface SelectProps extends FilterProps ,HTMLAttributes<HTMLSelectElement>{}

const Select = ({ name, options, onChange, ...restProps }: SelectProps) => {
  const optionElements = options.map((option) => {
    return (
      <option key={option.value} value={option.value}>
        {option.label}
      </option>
    );
  });

  return (
    <select {...restProps} className={styles.filter} name={name} onChange={onChange}>
      {optionElements}
    </select>
  );
};
```